<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TutorialPage</class>
 <widget class="QWidget" name="TutorialPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>530</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="tutorialPages">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="howToPlayLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;How To Play&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="page2Text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>This game was made to teach people about gear ratios, and what exactly they mean when discussing systems of gears.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="page2Image">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="text">
      <string>Gear Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="page2Text_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>134</y>
       <width>691</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A gear ratio refers to the ratio between the rotations of 2 gears in a system of gears.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="page2Text_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>247</y>
       <width>281</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>One of the gears is the driver gear, while the other is a driven gear. There can be multiple driven gears, as there can be more than 2 gears in a system of gears.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="page3Text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>These ratios can be calculated in a number of ways, the most common way being by looking at the number of teeth each gear has, and using them to calculate the gear ratio.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="page3Text_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>691</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>For example, given a 2-gear system where: 
- The driver gear has 10 teeth 
- The driven gear has 20 teeth 

The gear ratio would be 1:2, where the driven gear rotates once every time the driver gear rotates twice.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="howToPlayLabel_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Let's try the first level.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>420</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;If you need to re-read any information before &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;continuing, feel free to navigate back &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;by using the previous and next page buttons. &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Otherwise press Play to continue to the first level.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="nextPageButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousPageButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>530</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Page</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
